### PR TITLE
Added id to User.

### PR DESCRIPTION
**Pull request by BeeHiveJava**

### CODE REVIEW

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

File: api/src/main/java/nl/pts44/api/User.java [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
  * @author Lesley
  */
 public class User {
 
+    private final int id;
     private final String username;
 
-    public User(String username) {
-        this.username = username;
+    public User(int id, String username) {
```
Comment (TehMessiah) [inline api/src/main/java/nl/pts44/api/User.java]: BLOCKER Document this public constructor. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 1 issue
* ![BLOCKER][BLOCKER] 1 blocker

Watch the comments in this conversation to review them.

[BLOCKER]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER'

**Pull request merged**